src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -0,0 +1,34 @@
+package lotto.domain
+
+import lotto.util.*
+
+class Lotto(private val numbers: List<Int>) {
+    init {
+        require(numbers.size == 6)
+    }
+
+    override fun toString(): String {
+        return numbers.joinToString(prefix = "[", postfix = "]", separator = ", ") { it.toString() }
+    }
+
+    fun toList(): List<Int> {
+        return numbers
+    }
+
+    fun contains(number: Int): Boolean {
+        return numbers.contains(number)
+    }
+
+    companion object {
+        fun fromInput(inputNumbers: String): Lotto {
+            return fromList(inputNumbers.parseIntList())
+        }
+
+        fun fromList(numbers: List<Int>): Lotto {
+            return Lotto(numbers
+                    .validateCount(LottoStore.LOTTO_NUMBER_COUNT)
+                    .validateUnique(LottoStore.LOTTO_NUMBER_COUNT)
+                    .validateRange(LottoStore.LOTTO_MIN_NUMBER, LottoStore.LOTTO_MAX_NUMBER))
```

Comment:
제 생각에는 아래처럼 작성하는게 좀 더 깔끔한 것 같아요. 혹시 태준님 생각은 어떠신가요??
```suggestion 
            return numbers
                     .validateCount(LottoStore.LOTTO_NUMBER_COUNT)
                     .validateUnique(LottoStore.LOTTO_NUMBER_COUNT)
                     .validateRange(LottoStore.LOTTO_MIN_NUMBER, LottoStore.LOTTO_MAX_NUMBER)
                     .run { Lotto(this) }
```

Reply:
이렇게 하니까 더 깔끔하게 직관적인 것 같습니다. 자세한 피드백 감사드려요!